src/components/Footer.vue

```diff
@@ -45,4 +45,9 @@
 export default {
   name: 'Footer',
 }
-</script>
\ No newline at end of file
+</script>
+<style lang="scss">
```

Comment:
You should not do this, because you will be affecting all h5 in the app, and we don't want that, add the `scoped` attribute, so all the CSS created here will only affect this component
```suggestion
<style lang="scss" scoped>
```